components/chat_widget/src/components/ocs-chat/ocs-chat.css

```
@@ -400,6 +408,15 @@
     @apply shadow-2xl cursor-grabbing;
   }
 
+  .chat-window-kiosk {
+    @apply absolute inset-0 w-full h-full overflow-hidden flex flex-col;
+    border-radius: 0;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)